TheTofuWarriors/TofuWarrior/Startup.cs

```diff
@@ -28,11 +29,9 @@ public void ConfigureServices(IServiceCollection services)
         {
 
             services.AddControllers();
-            services.AddSwaggerGen(c =>
-            {
-                c.SwaggerDoc("v1", new OpenApiInfo { Title = "TofuWarrior", Version = "v1" });
-            });
-        }
+           
+            var cs = Configuration.GetConnectionString("Default");
+            services.AddDBContext<TheTofuWarriorsDBContext>
```

Comment:
Naima, the code doesn't seem to be building in the pipeline. I think you're just missing some parenthesis here on line 34 Startup.cs. If it ran in your local environment, then you can ignore this, but please make sure you're able to build and run the webApi before you make a Pull Request.